mainapp/views.py

```diff
@@ -58,7 +58,7 @@ def book_recommendations(request):
     random.shuffle(user_ratings)
     best_user_ratings = sorted(user_ratings, key=operator.attrgetter('bookrating'), reverse=True)
 
-    if len(best_user_ratings) < 4:
+    if len(best_user_ratings) < 5:
```

Comment:
You need to change this back, even if user has `5` ratings it will not pass

Reply:
Okey, I will revert back this change